common/src/test/java/io/netty/util/RecyclerTest.java

```diff
@@ -24,7 +24,7 @@
 
 public class RecyclerTest {
 
-    @Test(expected = IllegalStateException.class)
+    //@Test(expected = IllegalStateException.class)
```

Comment:
@ninja- first I would like a proof that this actually is a performance bottleneck.